sw/airborne/modules/optical_flow/px4flow_i2c.c

```diff
@@ -267,6 +267,7 @@ void px4flow_i2c_downlink(void)
   uint8_t id = PX4FLOW_I2C_ID;
 
   float timestamp = get_sys_time_float();
+  uint8_t distance_quality = 0;
 
 #if REQUEST_INT_FRAME
   int16_t flow_x = px4flow.i2c_int_frame.pixel_flow_x_integral;
```

Comment:
since fields are passed by pointer, it is important that the types are correct, so here `flow_x` and `flow_y` must be `int32_t` as well.